### ISSUE TITLE
Add a ICESat-2 L4 Monthly Gridded Sea Ice Thickness, Version 1 to the API 

### ISSUE THREAD

**Issue by abarciauskas-bgse** [NASA-IMPACT/cloud-optimized-data-pipelines#108]
We need to add this dataset to the API to support the trilateral dashboard cryospheric story, and presumably the NASA cryospheric thematic area in the NASA dashboard.

## Identify the dataset and what the processing needs are

- [x] Identify dataset and where it will be accessed from. Check it's a good source with science team. Ask about specific variables and required spatial and temporal extent. Note most datasets will require back processing (e.g. generating cloud-optimized data for historical data).

https://nsidc.org/data/IS2SITMOGR4/versions/1

There are only 22 files since it is a monthly dataset.
 
- [ ] The dataset is ongoing monthly so we need to design and construct the forward-processing workflow.

We may create a separate ticket for this. Since there are only 22 files we can probably start by doing the conversion locally but will want to deploy in a pipeline that runs the conversion and publishes a new product every month.
  
- [x] Create COG conversion code and verify the COG output with a data product expert (for example, someone at the DAAC which hosts the native format) by sharing in a visual interface.

There is some code here which is awaiting validation by Alessandro Scremin (EO Expert System Engineer)

https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/blob/ab/icesat2-IS2SITMOGR4-conversion/dataset-workflows/IS2SITMOGR4/IS2SITMOGR4-to-COG.ipynb

## Design the metadata and publish to the Dev API

- [x] Review conventions for generating STAC collection and item metadata:
   - Collections: https://github.com/NASA-IMPACT/delta-backend/issues/29 and STAC version 1.0 specification for collections
   - Items: https://github.com/NASA-IMPACT/delta-backend/issues/28 and STAC version 1.0 specification for items
   - NOTE: The delta-backend instructions are specific to datasets for the climate dashboard, however not all datasets are going to be a part of the visual layers for the dashboard so I believe you can ignore the instructions that are specific to "dashboard" extension, "item_assets" in the collection and "cog_default" asset type in the item
  
- [x] After reviewing the STAC documentation for collections and items and reviewing existing scripts for generating collection metadata (generally with SQL) and item metadata, generate or reuse scripts for your collection and a few items to publish to the testing API. There is some documentation and examples for how to generate a pipeline or otherwise document your dataset workflow in https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines. We would like to maintain the scripts folks are using to publish datasets in that repo so we can easily re-run those datasets ingest and publish workflows if necessary.

- [x] If necessary, request access and credentials to the dev database and ingest and publish to the Dev API. Submit a PR with the manual or CDK scripts used to run the workflow to publish to the Dev API and include links to the published datasets in the Dev API

## Publish to the Staging API

- [x] Once the PR is approved, we can merge and publish those datasets to the Staging API

**Comment by abarciauskas-bgse** [comment]
Alessandro said the new file looks good but only the date is required for the filenaming - in other words we should include "2021-01-01" in the filename but don't need the time, for example: 2021-01-01T00:00:00.000Z

**Comment by abarciauskas-bgse** [comment]
Anca reached out to ask about the progress on this dataset last week over slack so I think it is a high priority.

**Comment by slesaad** [comment]
@abarciauskas-bgse the linked dataset is version 1, but the most recent version is 2. Should we go with version 2 instead?

**Comment by abarciauskas-bgse** [comment]
Likely, checking with Kaitlin Harbeck. Thanks for noticing this!

**Comment by abarciauskas-bgse** [comment]
@slesaad yes can we publish both projections? I think if its the same data in different projections we might just have items with 2 assets (one asset for each projection for each source NetCDF), what do you think? cc @anayeaye

**Comment by abarciauskas-bgse** [comment]
@slesaad since we haven't heard any additional needs from the ESA EOX team I think we can close this
